pwndbg/disasm/x86.py

```diff
@@ -62,7 +61,7 @@ def memory(self, instruction, op):
     def memory_sz(self, instruction, op):
         arith = False
         segment = op.mem.segment
-        disp = op.value.mem.disp
+        # disp = op.value.mem.disp
```

Comment:
Maybe that should be `disp = op.mem.disp` and used later on?

This should probably be:
```py

        if disp != 0:
            if arith:
                sz += " - " if disp < 0 else " + "
            sz += f"{abs(disp):#x}"
```

I also wonder where is it displayed, because maybe "+" and space afterwards is irrelevant? If it were, we could just do:
```py
if disp != 0:
    sz += f"{disp:#x}"
```
but feel free to not investigate this in this PR

Reply:
Oh and when we are at it, we can change the:
```py
        sz = f"[{sz}]"
        return sz
```
to just `return f"[{sz}]"`